src/motion_capture.cc

```diff
@@ -215,16 +215,42 @@ void MotionCapturePlugin::OnUpdate(const gazebo::common::UpdateInfo &_info)
     root_transform(2,3) = shift_pos_.Z();
     // Recursively calculate cartesian joint positions.
     CalculateJointPos(node_pose, root_transform, this->root_node_, this->joint_positions_);
-    custom_robot_msgs::PositionsHeadered joint_pos_msg;
+    concert_msgs::Humans joint_pos_msg;
+    concert_msgs::Human3D human1;
```

Comment:
this name is not descriptive enough. How about
```
concert_msgs::Humans human_measurement_msg;
concert_msgs::Human3D human_pose_msg;
```

Reply:
Fixed with https://github.com/manuelvogel12/human-gazebo/commit/784d761426e586b9f72af9e7cfb75aba7cf350c1. I called it human**s**_measurement_msg instead